src/sentry/web/frontend/accounts.py

```diff
@@ -119,11 +120,12 @@ def start_confirm_email(request):
     has_unverified_emails = request.user.has_unverified_emails()
     if has_unverified_emails:
         request.user.send_confirm_emails()
-        msg = _('A verification email has been sent to %s.') % request.user.email
+        unverified_emails = request.user.get_unverified_emails()
+        msg = _('A verification email has been sent to %s.') % (', ').join([email.email for email in unverified_emails])
     else:
         msg = _('Your email (%s) has already been verified.') % request.user.email
     messages.add_message(request, messages.SUCCESS, msg)
-    return HttpResponseRedirect(reverse('sentry-account-settings'))
+    return HttpResponseRedirect(reverse('sentry-account-settings-emails'))
```

Comment:
FYI, these are causing [ConfirmEmailSendTest](https://github.com/getsentry/sentry/blob/master/tests/sentry/web/frontend/accounts/tests.py#L276) to [fail](https://travis-ci.org/getsentry/sentry/jobs/159677674#L926). Personally, I think this is the right move, so probably should update that test.